File: Aplicacao_cliente/SilentManager/app/src/main/java/com/silent_manager/g29/silent_manager/data_layer/models/Location.kt
package com.silent_manager.g29.silent_manager.data_layer.models

import android.arch.persistence.room.Entity
import com.fasterxml.jackson.annotation.JsonProperty
import java.io.Serializable

@Entity(tableName = "Location", primaryKeys = arrayOf("id"))
data class Location(
    @JsonProperty("locationId")
    val id: Int?,
    @JsonProperty("latitude")
    val latitude: Double?,
    @JsonProperty("longitude")
    val longitude: Double?,
    @JsonProperty("address")
    val address: String?
) : Serializable

File: Servidor/SilentManagerApiServer/Models/DB/Location.cs
using System;
using System.Collections.Generic;

namespace SilentManagerApiServer.Models.DB
{
    public partial class Location
    {
        public Location()
        {
            Event = new HashSet<Event>();
        }

        public int LocationId { get; set; }
        public double? Latitude { get; set; }
        public double? Longitude { get; set; }
        public string Address { get; set; }

        public virtual ICollection<Event> Event { get; set; }
    }
}
